sql/src/main/java/org/apache/druid/sql/calcite/schema/SystemSchema.java

```diff
@@ -159,6 +160,7 @@
       .add("tier", ValueType.STRING)
       .add("curr_size", ValueType.LONG)
       .add("max_size", ValueType.LONG)
+      .add("is_leader", ValueType.LONG)
```

Comment:
not sure why this is a LONG type variable and not STRING type which could have values "true", "false" and "not-applicable" ?

Reply:
I guess it's to be consistent with other boolean types such as `is_published` in the segments table. It is useful for those boolean columns in the segments table to be long so that we can easily compute sums of them. For this column, I think it doesn't have to be long, but I don't mind either.

Reply:
Yeah, what @jihoonson was my reasoning for using long, to be consistent with other boolean-ish columns, but i'm not super attached to it and can change if that is better

Reply:
thanks for explaining, in the big scheme of things it hardly matters... consider it a nit